UnityProject/Assets/Scripts/Closets/ClosetPlayerHandler.cs

```diff
@@ -38,6 +39,17 @@ public void Init(ClosetControl closetCtrl)
 
 		private void Update()
 		{
+			if (PlayerManager.LocalPlayerScript.playerNetworkActions.isGhost)
+			{
+				Camera2DFollow.followControl.damping = 0.0f;
+				return;
+			}
+			else
+			{
+				// In case we re-enter corpse
+				Camera2DFollow.followControl.damping = 0.2f;
+			}
+			
```

Comment:
What happens if LocalPlayerScript is null (When there is a few frames of spawning new player) and what happens if Camera2DFollow.followControl.damping needs to be set to something different from another object in a different scenario? This condition in all of the closets Update loops will override that. I would like you to come up with a better solution to changing the damping if a player turns into a ghost. You could just always set the damping property once whenever a player turns into a ghost no matter if they are in a closet or not (maybe in the local client of HealthBehaviour) then you can get rid of this condition.